### PR TITLE
added getNumCardsReviewedByDay action

### PR DESCRIPTION
**Pull request by qharscoet**
Might add options to filter by decks or something but for now seems to be usable

### CODE REVIEW

File: actions/statistics.md [hunk 1]
```diff
@@ -20,6 +20,29 @@
...
+        "action": "getNumCardsReviewedByDay",
+        "version": 6,
+    }
+    ```
+
+    *Sample result*:
+    ```json
+    {
+        "result":  [
+            ['2021-02-28', 124],
```
Comment (FooSoft) [inline actions/statistics.md:39]: These are single quotes, so it's not valid JSON

File: plugin/__init__.py [hunk 2]
```diff
@@ -337,6 +337,11 @@ def multi(self, actions):
     def getNumCardsReviewedToday(self):
         return self.database().scalar('select count() from revlog where id > ?', (self.scheduler().dayCutoff - 86400) * 1000)
 
+    @util.api()
+    def getNumCardsReviewedByDay(self):
+        return self.database().all("select date(id/1000 - ?, 'unixepoch', 'localtime') as day, count() from revlog group by day order by day desc",
```
Comment (FooSoft) [inline plugin/__init__.py:342]: Single quotes in python strings please

**Review by FooSoft — CHANGES_REQUESTED** [review]

**Comment by FooSoft** [comment]
Please also add some basic tests to make sure that we are at least calling these APIs.

**Comment by qharscoet** [comment]
Indeed, sorry for the wrong quotes.
Also added basic test inspired by the ones for the other stats actions.

**Comment by FooSoft** [comment]
Looks good, thanks!

**Pull request merged**